.github/workflows/fips-compliance.yml

```
@@ -0,0 +1,55 @@
+      run: |
+        docker build -f Dockerfile.redhat \
+          -t kube-auth-proxy:fips-test \
+          --build-arg VERSION=test \
+          --build-arg BUILDPLATFORM=linux/amd64 \
+          .
+    
+    - name: Setup Go for check-payload
+      uses: actions/setup-go@v5
+      with:
+        go-version: '1.21'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)